Source/WebCore/Modules/url-pattern/URLPatternCanonical.cpp

```diff
@@ -162,8 +163,13 @@ ExceptionOr<String> canonicalizePort(StringView portValue, const std::optional<S
 
     URL dummyURL(dummyURLCharacters);
 
-    if (protocolValue)
+    if (protocolValue) {
+        auto parsedPort = parseInteger<uint16_t>(portValue);
+        if (!parsedPort || isDefaultPortForProtocol(*parsedPort, *protocolValue))
+            return String { ""_s };
```

Comment:
```
URLPatternCanonical.cpp:169:20: error: no viable conversion from returned value of type 'const String' to function return type 'ExceptionOr<String>'
  169 |             return emptyString(); // String { ""_s };
  
  

```

It tried that initially but I get this complaint that emptyString() is const, so it can't be returned.

Reply:
Maybe `return { emptyString() };` or `return ExceptionOr<String> { emptyString() };`

Reply:
Also, remember to update this